spec/slack-gamebot/commands/taunt_spec.rb

```
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe SlackGamebot::Commands::Taunt, vcr: { cassette_name: 'user_info' } do
+  let!(:team) { Fabricate(:team) }
+  let(:app) { SlackGamebot::Server.new(team: team) }
+  let(:client) { app.send(:client) }
+  let(:user) { Fabricate(:user, user_name: 'username') }
+  let(:opponent) { Fabricate(:user) }
+  it 'creates a singles taunt by user name' do
+    expect do
```

This `expect` is unnecessary, and actually hides the failure from this spec.